src/models/DownloadLimitAction.js

```diff
@@ -58,7 +58,15 @@ export default class DownloadLimitAction {
 		return {
 			'update:value': debounce(async (limit) => {
 				this._store.loading = true
-				await setDownloadLimit(this._store.token, limit)
+
+				// If the value is not correct, let's remove the limit
+				if (!limit || limit < 1 || typeof limit !== 'number') {
```

Comment:
not sure if entering anything other then a number should also delete the count?

Reply:
On it, yeah, made a logic mistake ;)

Reply:
Fixed